src/Components/Header/Logo/Logo.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const Logo=()=> {
+  return (
+    <div className="logo">
+      <span className="logo-icon material-icons">shopping_cart</span> PrimeCart
```

Comment:
Material Icons can be added directly from the [@mui/icons-material](https://www.npmjs.com/package/@mui/icons-material) package. 
You can have a look at the docs - [https://mui.com/components/icons/](https://mui.com/components/icons/)